io/zenoh-transport/src/unicast/lowlatency/transport.rs

```diff
@@ -187,6 +188,10 @@ impl TransportUnicastTrait for TransportUnicastLowlatency {
         self.config.zid
     }
 
+    fn get_auth_ids(&self) -> Vec<AuthId> {
+        vec![]
```

Comment:
@Mallets Does this seem correct to you?

Reply:
No, it seems an incomplete implementation.